Resources/views/Collector/db.html.twig

```diff
@@ -409,5 +410,81 @@
+                    <span style="display: none">Shrink query</span>
+                    <span id="smallcode-{{ i }}-{{ loop.parent.loop.index }}">
+                        {{ query.sql|doctrine_minify_query }}
+                    </span>
+                </div>
+                <div id="code-{{ i }}-{{ loop.parent.loop.index }}" style="display: none;">
+                    {{ query.sql|doctrine_pretty_query(i, loop.parent.loop.index) }}
+                </div>
+                <span id="original-query-{{ i }}-{{ loop.parent.loop.index }}" style="display: none;">
+                    {{ query.sql|doctrine_replace_query_parameters(query.params) }}
+                </span>
+                <small>
+                    <strong>Parameters</strong>: {{ query.params|yaml_encode }} <br />
+                    [<span id="expandParams-{{ i }}-{{ loop.parent.loop.index }}" onclick="javascript:toggleRunnableQuery(this);" data-target-id="original-query-{{ i }}-{{ loop.parent.loop.index }}" style="cursor: pointer;">Display runnable query</span>]<br/>
+                </small>
+
+                {% if query.explainable %}
+                    [<a href="{{ path('_profiler', {'panel': 'db', 'token': token, 'page': 'explain', 'connection': connection, 'query': i}) }}" onclick="return explain(this);" style="text-decoration: none;" title="Explains the query" data-target-id="explain-{{ i }}-{{ loop.parent.loop.index }}" >
+                    <img alt="+" src="data:image/gif;base64,R0lGODlhEgASAMQTANft99/v+Ga44bHb8ITG52S44dXs9+z1+uPx+YvK6WC24G+944/M6W28443L6dnu+Ge54v/+/l614P///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAABMALAAAAAASABIAQAVS4DQBTiOd6LkwgJgeUSzHSDoNaZ4PU6FLgYBA5/vFID/DbylRGiNIZu74I0h1hNsVxbNuUV4d9SsZM2EzWe1qThVzwWFOAFCQFa1RQq6DJB4iIQA7" style="display: inline; width: 12px; height: 12px;" />
+                    <img alt="-" src="data:image/gif;base64,R0lGODlhEgASAMQSANft94TG57Hb8GS44ez1+mC24IvK6ePx+Wa44dXs92+942e54o3L6W2844/M6dnu+P/+/l614P///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAABIALAAAAAASABIAQAVCoCQBTBOd6Kk4gJhGBCTPxysJb44K0qD/ER/wlxjmisZkMqBEBW5NHrMZmVKvv9hMVsO+hE0EoNAstEYGxG9heIhCADs=" style="display: none; width: 12px; height: 12px;" />
+                    <span style="vertical-align:top">Explain query</span>
+                </a>]
+                {% else %}
+                    This query cannot be explained
+                {% endif %}
+
+                {% if query.explainable %}
+                    <div id="explain-{{ i }}-{{ loop.parent.loop.index }}" class="loading"></div>
+                {% endif %}
+            </td>
+        </tr>
+    {% endfor %}
+{% endblock %}
+
+{% block query_table_v2 %}
+    {% for i, query in queries %}
+        <tr id="queryNo-{{ i }}-{{ loop.parent.loop.index }}">
+            <td>{{ loop.index }}</td>
+            <td class="nowrap">{{ '%0.2f'|format(query.executionMS * 1000) }} ms</td>
```

Comment:
My guess is that the explicit `&nbsp;` was intended to solve the same problem that we now solve with the `nowrap` CSS class.